app/src/main/java/grinnell/appdev/edu/lyles/fragments/ScheduleFragment.java

```diff
@@ -1,22 +1,82 @@
 package grinnell.appdev.edu.lyles.fragments;
 
+import android.app.ListFragment;
+import android.database.DataSetObserver;
 import android.os.Bundle;
 import android.support.annotation.Nullable;
 import android.support.v4.app.Fragment;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.ListAdapter;
+import android.widget.ListView;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+
+import java.util.ArrayList;
 
 import grinnell.appdev.edu.lyles.R;
+import grinnell.appdev.edu.lyles.ScheduleItem;
+import grinnell.appdev.edu.lyles.ScheduleTabAdapter;
 
 /**
  * Created by Mattori on 5/9/16.
  */
 public class ScheduleFragment extends Fragment {
 
+
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
```

Comment:
I like that you are testing things but we should not have tests here. You should test (using Instrumented tests) this class in the `androidTest` package. Check out [Espresso](https://developer.android.com/training/testing/ui-testing/espresso-testing.html) to see how to go about this.